source/unified-test-format/unified-test-format.md

```diff
@@ -2939,12 +2939,19 @@ command to create the collection with the specified options. The test runner MUS
 topology is sharded, the test runner SHOULD use a single mongos for handling [initialData](#initialData) to avoid
 possible runtime errors.
 
+After dropping, creating and populating initial data for all collections, the test runner MUST get the latest cluster
+time from the internal MongoClient and store it for later use when creating session entities. A simple way to get the
+latest cluster time is to execute a `ping` command and get the value of the `$clusterTime` field in the reply.
```

Comment:
I thought that the internal MongoClient is supposed to connect to a single mongos.

Reply:
Per Executing a Test

> If the topology is sharded, the test runner SHOULD use a single mongos for handling initialData to avoid possible runtime errors.

Given that, would a driver still need an explicit session object?  So long as a MongoClient connected to the same single mongos is used to drop, create, populate, and get cluster time, that should be sufficient.

Reply:
The internal client being connected to a single mongos is not guaranteed, albeit likely for most implementations. Earlier in Initializing the Test Runner we have:

> If this (internal) MongoClient would connect multiple mongos nodes and the driver does not provide a way to target operations to specific servers, the test runner MAY construct internal MongoClients for each mongos.

I'll ensure that both the `ping` and explicit session approaches are discussed in the text.